cs/src/core/Index/FASTER/FASTER.cs

```diff
@@ -23,8 +26,16 @@ public unsafe partial class FasterKV<Key, Value, Input, Output, Context, Functio
         private readonly bool CopyReadsToTail = false;
         private readonly bool FoldOverSnapshot = false;
         private readonly int sectorSize;
-
         private readonly bool WriteDefaultOnDelete = false;
+        private bool RelaxedCPR = false;
+
+        /// <summary>
+        /// Use relaxed version of CPR, where ops pending I/O
+        /// are not part of CPR checkpoint. This mode allows
+        /// us to eliminate the WAIT_PENDING phase, and allows
+        /// sessions to be suspended. Do not modify during checkpointing.
+        /// </summary>
+        public void UseRelaxedCPR() => RelaxedCPR = true;
```

Comment:
Is there a way for us to ensure that this is not modified during checkpointing?

Will people want to modify this at all for a single Faster instance?